modules/http_push.c

```diff
@@ -96,6 +107,14 @@ static int nw_http_push_start_acquire_data(struct nodewatcher_module *module,
           case CURLE_SSL_PINNEDPUBKEYNOTMATCH:
 #endif
           case CURLE_PEER_FAILED_VERIFICATION: push_result = "peer_verify_error"; break;
+          case CURLE_SSL_CONNECT_ERROR:
+            push_result = "ssl_connect_error";
+            if (strlen(errbuf)) {
+              syslog(LOG_WARNING, "buf: %s", errbuf);
```

Comment:
Can you change the warning to be something more descriptive? E.g., `http-push: SSL connect error: %s`?